src/custom/pages/Claim/Countdown.tsx

```diff
@@ -0,0 +1,67 @@
+// Sort of a mod of but not quite from src/pages/Earn/Countdown.tsx
+import { useEffect, useState } from 'react'
+
+const MINUTE = 60
```

Comment:
`ms` lib?

Reply:
Just copied over man, don't wanna spend time "thinking" about this part =P

Reply:
lol /shrug ok